store/tikv/2pc.go

```diff
@@ -621,7 +621,8 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) error {
 	if err != nil {
 		if undeterminedErr := c.getUndeterminedErr(); undeterminedErr != nil {
 			log.Warnf("con:%d 2PC commit result undetermined, err: %v, rpcErr: %v, tid: %v", c.connID, err, undeterminedErr, c.startTS)
-			err = errors.Wrap(err, terror.ErrResultUndetermined)
+			log.Error(err)
+			err = errors.Trace(terror.ErrResultUndetermined)
```

Comment:
can we remove the `errors.Trace()`?

Reply:
for this point, maybe better keep `Trace` beacause ErrResultUndetermined is reusable global variable without stack.